playwright.config.ts

```diff
@@ -38,9 +44,10 @@ export default defineConfig({
 		...(DEV && { headless: false })
 	},
 	projects: [
+		/* Test against desktop browsers. */
 		{
 			name: 'Google Chrome',
-			use: { ...devices['Desktop Chrome'], channel: 'chrome' }
+			use: { ...devices['Desktop Chrome'] }
```

Comment:
Is this change related to this PR?

Reply:
Eh, I changed this some time ago, and I am pretty sure it "improves" the quality of the screenshots. So when I realized it got changed back I reverted it to what I did before.

Reply:
Does it improves the quality because we get a more recent or older chrome by using another channel? And if so, should we pin the version of chrome we are actually using?

Reply:
By defining a "channel" for a browser you make sure that you use a specific version, installed on the device that runs the test. By not using a "branded browser" we use playwrights device library and open source browsers. With that we remove the potential danger of having browser issues because not everything is installed on the github runner.

Its not a complete wow effect but I feel like if we are using playwright and they offer a wide range of devices, why not just use what they have to offer.

Reply:
The ci will be a bit slower I guess because it will have to fetch Chrome. Alright, not sure how it's related but, indeed if it improves the screenshots, all good. Thanks for the explanation.